### PR TITLE
Fix compilation on Termux

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Fix io_shm.c compilation and bundled libuv.

**Test plan**

Compile rizin on Termux with everything bundled.

**Closing issues**

...

### CODE REVIEW

File: librz/include/rz_userconf.h.in [hunk 1]
```diff
@@ -59,7 +63,7 @@
 #define RZ_DATDIR  "@DATADIR@"
 #define RZ_WWWROOT "@WWWROOT@"
 
-#define RZ_BINDIR_DEPTH  @BINDIR_DEPTH@
+#define RZ_BINDIR_DEPTH @BINDIR_DEPTH @
```
Comment (wargio) [inline librz/include/rz_userconf.h.in]: this is 100% wrong.

Comment (ret2libc) [inline librz/include/rz_userconf.h.in]:
```suggestion
#define RZ_BINDIR_DEPTH @BINDIR_DEPTH@
```

File: librz/include/rz_userconf.h.in [hunk 2]
```diff
@@ -59,7 +63,7 @@
 #define RZ_DATDIR  "@DATADIR@"
 #define RZ_WWWROOT "@WWWROOT@"
 
-#define RZ_BINDIR_DEPTH  @BINDIR_DEPTH@
+#define RZ_BINDIR_DEPTH @BINDIR_DEPTH@
```
Comment (ret2libc) [inline librz/include/rz_userconf.h.in]:
```suggestion
#define RZ_BINDIR_DEPTH  @BINDIR_DEPTH@
```

**Review by XVilka — APPROVED** [review]
As long as it works.
@devnexen could you please help us to check on Haiku OS, DragonFly, etc, if possible?

**Comment by devnexen** [comment]
nearly flawlessly for Haiku but that s not rizin fault, maybe meson version bugged, for the rzwindkd I had to modify manually the meson build otherwise it complains there is no compiler but I do not know meson well enough if this is capable to propagate variables among sub projects like cmake for example.

**Comment by wargio** [comment]
can you copy and paste here the changes and the error message?

**Comment by devnexen** [comment]
oh I just added the typical
`cc = meson.get_compiler('c')`

because `subprojects/rzwinkd/meson.build:22:2: ERROR: Unknown variable "cc".`

**Comment by wargio** [comment]
@ret2libc can you add that to the file?

**Comment by ret2libc** [comment]
> @ret2libc can you add that to the file?

done!

**Review by wargio — APPROVED** [review]

**Pull request merged**